examples/features/sub-modules/wlp-jakarta/pom.xml

```diff
@@ -0,0 +1,145 @@
+<!--
+~ Licensed to the Apache Software Foundation (ASF) under one or more
+~ contributor license agreements. See the NOTICE file distributed with
+~ this work for additional information regarding copyright ownership.
+~ The ASF licenses this file to You under the Apache License, Version 2.0
+~ (the "License"); you may not use this file except in compliance with
+~ the License. You may obtain a copy of the License at
+~
+~     http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.apache.activemq.examples.modules</groupId>
+      <artifactId>broker-modules</artifactId>
+      <version>2.22.0-SNAPSHOT</version>
+   </parent>
+
+   <artifactId>wlp-jakarta-resource-adapter-sample</artifactId>
+   <packaging>war</packaging>
+   <name>Artemis Websphere Liberty RA Jakarta Example</name>
+
+   <properties>
+      <activemq.basedir>${project.basedir}/../../../..</activemq.basedir>
+      <jakarta.jms-api.version>3.0.0</jakarta.jms-api.version>
+      <jakarta.annotation-api.version>2.0.0</jakarta.annotation-api.version>
+      <jakarta.ws.rs-api.version>3.0.0</jakarta.ws.rs-api.version>
+      <jakarta.ejb-api.version>4.0.0</jakarta.ejb-api.version>
```

Comment:
Having every jakarta related module hard code a specific version of various specs will be quite annoying later. I have pushed some changes to main addressing this for existing stuff, see https://github.com/apache/activemq-artemis/commit/6eb78a26348db54164128c32c7321f1a8ae23c35. You should use the new props I added where appropriate, and follow the same approach to introduce other ones for the other deps this PR uses.